getty.go

```diff
@@ -171,7 +171,9 @@ type Session interface {
 
 	// the Writer will invoke this function. Pls attention that if timeout is less than 0, WritePkg will send @pkg asap.
 	// for udp session, the first parameter should be UDPContext.
-	WritePkg(pkg interface{}, timeout time.Duration) error
+	// total: need send total bytes
```

Comment:
// WritePkg: encode package @pkg to bytes steam and send them out
//
// totalBytesLength: @pkg stream bytes length after encoding @pkg.
// sendBytesLength: stream bytes length that sent out successfully.
// error: send out error from system

Reply:
done